src/puppeteer-provider.ts

```diff
@@ -366,7 +360,12 @@ export class PuppeteerProvider {
     const handler = debugCode ?
       (done: IDone) => {
         jobdebug(`${job.id}: Starting debugger sandbox.`);
-        const doneOnce = _.once(done);
+        const doneOnce = _.once((err?: Error) => {
```

Comment:
I think this instance, and the two duplicates above, should be encapsulated in a method.